aptos-move/framework/aptos-names/sources/config.move

```diff
@@ -60,6 +64,11 @@ module aptos_names::config {
         set_tokendata_description(framework, string::utf8(b"This is an official Aptos Foundation Name Service Name"));
         set_tokendata_url_prefix(framework, string::utf8(b"https://aptosnames.com/api/v1/metadata/"));
 
+        // TODO: SET THIS TO SOMETHING REAL
+        let public_key: vector<u8> = vector<u8>[78, 223, 79, 162, 141, 187, 221, 61, 167, 178, 113, 30, 108, 153, 98, 132, 30, 250, 154, 102, 169, 41, 70, 119, 66, 234, 162, 31, 95, 133, 133, 111];
+        set_admin_public_key(framework, public_key);
```

Comment:
Is this expected to be different from the admin address, which is already a multisig scheme based address/auth key?